### DISCUSSION TITLE
config arguement lagged_features use?

### DISCUSSION THREAD

**Discussion by Meelisha** [neuralhydrology/neuralhydrology#70] [Q&A] [answered]
Can we use the above configuration argument to use observed streamflow from previous timesteps as input in the training process? If not, is there a way we can use the autoregression modelling approach along with the LSTM? I might have used some terminologies incorrectly, so apologies for that. Any help would be appreciated. Thank you!

**Comment by kratzert** [comment, marked as answer]
Hi @Meelisha 

yes, this use case was exactly the reason why we implemented the lagged_features logic. Have you tried it?

**Comment by Meelisha** [comment]
Hi @kratzert ,

Yes I did try with 1,2 and 3 days lag data of streamflow. I could definitely see improvement in the NSE values for individual basin as well as the overall average of NSE. However, the improvement wasn't considerable. I will try different configurations to be sure. Thanks!

**Reply by arasburakaslan** [reply]
Hello Meelisha, 

Did you add this argument to the config file by yourself? I dont have some of the arguments that are mentioned in the documentation so I was thinking if I should just add them myself or there is a new version of the file that I missed.

**Reply by Meelisha** [reply]
Hello @arasburakaslan ,

Yes I did add the argument myself. I don't think the lagged_features are included in the tutorial files (if that is what you mean by the new version of file).
